azure-ts-appservice/README.md

```diff
@@ -28,10 +28,9 @@ with App Service.
 1. Define SQL Server password (make it complex enough to satisfy Azure policy):
 
     ```
-    pulumi config set <stack>:sqlPassword <value>
+    pulumi config set --secret <stack>:sqlPassword <value>
```

Comment:
Note that we should be able to remove the `<stack>` part here as well.

Reply:
it needs to be `pulumi config set --secret azure-appservice:sqlPassword <value>` right?

Reply:
You can leave that off and it will default to the project-scoped config.